### PR TITLE
Multimodal_RNNs

### PR DESCRIPTION
**Pull request by faezehamjadi**

### CODE REVIEW

File: deep-learning/_posts/2018-06-01-MDRNN.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: review
+title:  "Multimodal Recurrent Neural Networks with Information Transfer Layers for Indoor Scene Labeling"
+tags:   "Deep Learning"
+author: Faezeh Amjadi
+pdf:    https://arxiv.org/pdf/1803.04687.pdf
+cite:
+authors: "Abrar H. Abdulnabi, Bing Shuai, Zhen Zuo, Lap-Pui Chau, Gang Wang"
+title:   " Indoor Scene Labeling"
```
Comment (jodoin) [inline deep-learning/_posts/2018-06-01-MDRNN.md]: Please put the complete title of the paper as well as the complete arXiv reference.

File: deep-learning/_posts/2018-06-01-MDRNN.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+layout: review
+title:  "Multimodal Recurrent Neural Networks with Information Transfer Layers for Indoor Scene Labeling"
+tags:   "Deep Learning"
```
Comment (jodoin) [inline deep-learning/_posts/2018-06-01-MDRNN.md]: add : RNN, CNN, segmentation

File: deep-learning/_posts/2018-06-01-MDRNN.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+### Description
+
+They propose a new method called Multimodal RNNs for RGB-D scene semantic segmentation to classify image pixels given two input sources(modalities): RGB color channels and Depth maps. 
+The model has two RNNs, one of which is assigned to learn the representations of a single
+input modality. To connect both RNN modalities and allow information fusion, information transfer layers is employed to select and transfer the relevant patterns from one modality to the other modality, and vice
+versa.
+
+Each modality employes a multidimensional recurrent neural network, The basic idea of (MDRNNs) is to replace the single recurrent connection found in standard recurrent networks with as many connections as there are
```
Comment (jodoin) [inline deep-learning/_posts/2018-06-01-MDRNN.md]: network.  The...

File: deep-learning/_posts/2018-06-01-MDRNN.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+spatiotemporal dimensions in the data, so at every point, the layer has already visited the points one step
+back along with every dimension. The hidden activations at these previous points are fed to the current
+point through recurrent connections, along with the input as shown in Fig(1). 
+
+
+<img src="/deep-learning/images/mdrnn/1.png" width="400">
+
+As you see, this RNN have just a direction, but we usually want surrounding context in all directions. The generalization of bidirectional networks with four hidden layer one starting in the top left and scanning down and right, one starting in the bottom left and scanning up and right,so all the hidden layers are connected to a single output layer, which therefore receives information about all surrounding context.
```
Comment (jodoin) [inline deep-learning/_posts/2018-06-01-MDRNN.md]: this RNN has 
one direction

File: deep-learning/_posts/2018-06-01-MDRNN.md (hunk 1 above)
Comment (faezehamjadi) [reply deep-learning/_posts/2018-06-01-MDRNN.md]: I made mistake about the name after I push it, I found.

**Pull request merged**